packages/tasks/src/hardware.ts

```diff
@@ -136,6 +136,10 @@ export const SKUS = {
 				tflops: 32.62,
 				memory: [24],
 			},
+			"V100": {
+				tflops: 130,
+				memory: [32],
```

Comment:
```suggestion
				memory: [32, 16],
```

There are variants with 16 GB of RAM: https://images.nvidia.com/content/technologies/volta/pdf/volta-v100-datasheet-update-us-1165301-r5.pdf